src/components/inputtextarea/InputTextArea.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const InputTextArea = ({label, placeholder}) => {
+  return(
+    <div className="input-group">
```

Comment:
This class can be named as `input__group`

Reply:
🆗